README.md

```diff
@@ -1,5 +1,5 @@
-# DPX Go SDK Version 0.0.3
-Go client library to interact with the various [DPX Service APIs]
+# Data Product Exchange Go SDK Version 0.0.3
+Go client library to interact with the various [Data Product Exchange Service APIs]
```

Comment:
```suggestion
Go client library to interact with the various Data Product Exchange Service APIs.
```
Remove the []'s since there is no category within the IBM Cloud catalog to link to.

Reply:
Updated.